src/SupraLib/Beamformer/RxBeamformerParameters.cpp

```diff
@@ -68,50 +85,106 @@ namespace supra
 
 	void RxBeamformerParameters::writeMetaDataForMock(std::string filename, std::shared_ptr<const USRawData> rawData) const
 	{
-		std::ofstream f(filename);
-		f << "rawDataMockMetadata v 3" << std::endl;
-		f << rawData->getNumElements() << " "
-			<< rawData->getElementLayout().x << " "
-			<< rawData->getElementLayout().y << " "
-			<< rawData->getNumReceivedChannels() << " "
-			<< rawData->getNumSamples() << " "
-			<< rawData->getNumScanlines() << " "
-			<< m_rxScanlineLayout.x << " "
-			<< m_rxScanlineLayout.y << " "
-			<< rawData->getImageProperties()->getDepth() << " "
-			<< rawData->getSamplingFrequency() << " "
-			<< m_rxNumDepths << " "
-			<< m_speedOfSoundMMperS << std::endl;
+		Json::Value jsonDoc;
```

Comment:
@waltsims Does anything on TUM side rely on being able to write the old "ascii" mock files?
With this, only JSON mocks can be written going forward, but the old ones can still be read.

Reply:
@goeblr No requirements for this feature on the TUM-side 👍